### ISSUE TITLE
Plugin doesn't work on Vault Docker image based on Alpine Linux

### ISSUE THREAD

**Issue by binlab** [LeSuisse/vault-gpg-plugin#28]
The plugin doesn't work in Vault Docker image based on Alpine Linux
```shell
/etc/vault/plugins # wget https://github.com/LeSuisse/vault-gpg-plugin/releases/download/v0.2.2/linux_amd64.zip
...
linux_amd64.zip      100% |********************************************************************************************************************************************************************| 7401k  0:00:00 ETA
/etc/vault/plugins # unzip linux_amd64.zip 
Archive:  linux_amd64.zip
  inflating: linux_amd64.sha256sum
  inflating: vault-gpg-plugin
/etc/vault/plugins # ls -la vault-gpg-plugin 
-rwxr-xr-x    1 root     root      16041188 Mar 23 16:02 vault-gpg-plugin
/etc/vault/plugins # ./vault-gpg-plugin 
sh: ./vault-gpg-plugin: not found
/etc/vault/plugins # 
```
It seems a problem with a build. Can you check, please?

**Comment by binlab** [comment]
And yes, the same error from Vault
```shell
vault_mount.gpg: Creating...

Error: error writing to Vault: Error making API request.

URL: POST https://vault.example.com/v1/sys/mounts/gpg
Code: 400. Errors:

* fork/exec /etc/vault/plugins/vault-gpg-plugin: no such file or directory

  on gpg.tf line 11, in resource "vault_mount" "gpg":
  11: resource "vault_mount" "gpg" {
```

**Comment by LeSuisse** [comment]
Hi,

Thanks for the report. The binaries are built on amd64 machine with a glibc. The build script does not explicitly disable cgo so the resulting linux_amd64.zip does not work with Alpine.

**Comment by binlab** [comment]
Oh, I see. Can I ask you to configure additional artifacts for images based on `musl` library (without `glibc`)

**Comment by binlab** [comment]
Already done - https://github.com/LeSuisse/vault-gpg-plugin/commit/cbe615162fc00de2bd3e1505c1f3ae1c48682c92
So fast, awesome! Thank you!

**Comment by LeSuisse** [comment]
New prebuilt binaries have been published under the v0.2.4 release.

https://github.com/LeSuisse/vault-gpg-plugin/releases/tag/v0.2.4

**Comment by binlab** [comment]
tested, now it works. Thanks!
